internal/functions/serve/serve.go

```diff
@@ -66,6 +66,13 @@ func Run(ctx context.Context, slug string, envFilePath string, verifyJWT bool, i
 			env = append(env, "VERIFY_JWT=false")
 		}
 
+		// mirror local DENO_* env variables to the Docker container.
+		env = append(env,
+			"DENO_TLS_CA_STORE="+os.Getenv("DENO_TLS_CA_STORE"),
+			"DENO_CERT="+os.Getenv("DENO_CERT"),
```

Comment:
don't we need to mount the cert into the deno container as well?

Reply:
hmm..yeah you're right. env variable alone won't be enough as the container has no access to cert store. Wonder if this can be solved with a network config on docker?